src/codeu/chat/DerbyStore.java

```diff
@@ -81,18 +82,18 @@ public DerbyStore() {
 			stmt = conn.createStatement();
 			
 			// Create the chat user table
-			stmt.execute("CREATE TABLE " + userTableName + "(id varchar(255), name varchar(255), password varchar(255), creation BIGINT)");
+			stmt.execute("CREATE TABLE " + userTableName + " (userid varchar(255), name varchar(255), password varchar(255), creation BIGINT, PRIMARY KEY (userid))");
```

Comment:
Name, for now, is guaranteed to be unique. That will be their username (so I should change the name of that column to username). Could I possibly have two primary keys?